include/modules/httpd.h

```diff
@@ -30,6 +30,60 @@
 #include <sstream>
 #include <map>
 
+class HTTPQueryParameters : public insp::flat_multimap<std::string, std::string>
```

Comment:
I can't imagine the methods in this class would enough use to justify their existence and they seem like a leaky abstraction considering how you can (apparently?) have multiple values for each key.

Reply:
The idea was to allow for easier access to simple values where grabbing the first value passed for a key is fine and allow for easy defaults